packages/manager/apps/dedicated/client/app/dedicated/server/bandwidth/private-order/private-order.routing.js

```diff
@@ -17,6 +17,10 @@ export default /* @ngInject */ ($stateProvider) => {
       translations: { value: ['.'], format: 'json' },
       resolve: {
         goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+        hasDefaultPaymentMethod: /* @ngInject */ (ovhPaymentMethod) =>
```

Comment:
I wonder if this information should be aggregated to user information through /configuration for instance as it is used quite a lot